### PR TITLE
Conversion to extension

### PR DESCRIPTION
**Pull request by ghost**
This PR turns the theme into an extension, allowing LESS, JS, and PHP to be used, and managed centrally

### CODE REVIEW

**Comment by ghost** [comment]
Also, if you need a website to display this theme, we might be able to provide hosting for you, as you can't use custom extensions with FreeFlarum....

**Review by ardacebi — APPROVED** [review]
LGTM.

**Comment by ardacebi** [comment]
@PartialDevAdmin I will merge this after getting reviews from @Nathan2023 and @valaki52, merge #75 and update the app.less file with the updated style file.

**Review by ardacebi — COMMENTED** [review]
Shouldn't we delete the material.less file on the main directory as we already imported the style to the app.less file? @PartialDevAdmin

**Review by ghost — APPROVED** [review]
LGTM.

**Comment by TurboProgramming** [comment]
There is an important issue with the admin panel because of PRs I merged so I'll wait for that to be fixed before merging the current style file to the updates app.less file.

**Comment by ardacebi** [comment]
Should be OK for now.

File: js/admin/Gulpfile.js
Comment (ardacebi) [inline js/admin/Gulpfile.js:5]: Is "extum" right? Shouldn't it be "Extum"?

File: js/admin/dist/extension.js
Comment (ardacebi) [inline js/admin/dist/extension.js:3]: Is "extum" right? Shouldn't it be "Extum"?

File: js/admin/dist/extension.js
Comment (ardacebi) [inline js/admin/dist/extension.js:13]: Is "extum" right? Shouldn't it be "Extum"?

File: js/forum/Gulpfile.js
Comment (ardacebi) [inline js/forum/Gulpfile.js:5]: Is "extum" right? Shouldn't it be "Extum"?

File: js/admin/src/main.js
Comment (ardacebi) [inline js/admin/src/main.js:3]: Is "extum" right? Shouldn't it be "Extum"?

File: js/forum/dist/extension.js
Comment (ardacebi) [inline js/forum/dist/extension.js:3]: Is "extum" right? Shouldn't it be "Extum"?

**Comment by ghost** [comment]
No, in extensions, we keep things lowercase, especially classes. Feel free to camelcase, but an uppercase Extum doesn't fit there

**Pull request merged**
